lib/routes/workflows.js

```diff
@@ -0,0 +1,196 @@
+          if (approval < 0) {
+            _.set(req.session, '404', {
+              message: config.workflows.messages.approved,
+              safe: `/${config.content.base}/${req.params.type}`,
+            });
+
+            return next();
+          }
+
+          // grab the the next step in approval workflow
+          const step = steps[approval];
+
+          return workflows.model().then(model => {
+            return content.only('workflow', values, model).then(merged => {
+              return content.form(merged, errors).then(form => {
+                res.render('content/approval', {
+                  title: `${config.content.actions.approve} :: ${req.params.type}`,
+                  form,
+                  action: `/${config.content.base}/${req.params.type.toLowerCase()}/${config.content.actions.approve}`,
+                  revision,
+                  data,
+                  type,
+                  config: config.content,
+                  step,
+                });
+              });
+            });
+          }).catch(error => {
+            next(error);
+          });
+        });
+    });
+
+    /*
+     * @name Save - Post to Workflows
+     * Save workflow to db
+     *
+     * @param {object} req - HTTP Request
+     * @param {object} res - HTTP Response
+     * @param {object} next - Express callback
+    */
+    app.post(`/${config.content.base}/:type/${config.content.actions.approve}`, (req, res, next) => {
+      const type = utils.singleItem('id', req.params.type.toLowerCase(), types);
+      const revision = _.get(req.session, `form.content.approve[${req.params.type.toLowerCase()}].revision`, {});
+      const workflow = utils.workflow(type, allflows, config);
+
+      // check type exists
+      if (!croutes.check.type(req, type)) {
+        return next();
+      }
+
+      // check workflow exists
+      if (!workflow) {
+        return next();
+      }
+
+      return workflows.model().then(model => {
+        return content.only('workflow', {}, model).then(merged => {
+          // validate audit entry
+          const validated = content.form.validate(req.body, merged);
+
+          if (validated === true) {
+            // grab the current workflow entries from the database
+            return database(`content-type--${type.id}`).select('*').where('revision', revision).then(rows => {
+              const rev = rows[0];
+              const audits = workflows.audits(rev, workflow, req);
+
+              return database(`content-type--${req.params.type.toLowerCase()}`)
+                .where('revision', '=', parseInt(revision, 10))
+                .update(audits);
+            }).then(() => {
+              _.unset(req.session, 'form.content.approve');
+              res.redirect(`/${config.content.base}/${req.params.type}`);
+            }).catch(e => {
+              next(e);
+            });
+          }
+
+          // eslint thinks the return above ruins this else
```

Comment:
It does. That rule says that if there is a return inside of an `if`, you shouldn't wrap the remaining `else` in an `if` statement